app/lib/screens/app_onboarding/app_outdated_screen.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
+import 'package:studyu_app/constants.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AppOutdatedScreen extends StatelessWidget {
+  const AppOutdatedScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final loc = AppLocalizations.of(context)!;
+    String? storeUrl;
+    IconData? storeIcon;
+    if (Platform.isAndroid) {
+      storeUrl = playStoreUrl;
+      storeIcon = MdiIcons.googlePlay;
+    } else if (Platform.isIOS) {
+      storeUrl = appstoreUrl;
+      storeIcon = MdiIcons.apple;
+    }
+    return Scaffold(
```

Comment:
Include an else to handle the web instance and other platforms we might support in the future. In that case do not show any url.

Reply:
The url will not be shown if the platform is not Android or iOS. So no change is needed.

Reply:
Other platforms will currently throw an exception. We should handle that:

```
======== Exception caught by widgets library =======================================================
The following UnsupportedError was thrown building AppOutdatedScreen(dirty, dependencies: [_LocalizationsScope-[GlobalKey#40f19]]):
Unsupported operation: Platform._operatingSystem

The relevant error-causing widget was: 
  AppOutdatedScreen AppOutdatedScreen:studyu/app/lib/routes.dart:66:56
When the exception was thrown, this was the stack: 
[...]
==> packages/studyu_app/screens/app_onboarding/app_outdated_screen.dart 17:18         build
```

Reply:
Fixed in https://github.com/hpi-studyu/studyu/pull/576/commits/40f39afe60293bfd62ea705a55f59012c6aa00e3